src/topology/metric_space/gromov_hausdorff_realized.lean

```diff
@@ -144,9 +144,10 @@ private lemma candidates_dist_bound  (fA : f ∈ candidates α β) :
 private lemma candidates_lipschitz_aux (fA : f ∈ candidates α β) :
   f (x, y) - f (z, t) ≤ 2 * max_var α β * dist (x, y) (z, t) :=
 calc
-  f (x, y) - f(z, t) ≤ f (x, t) + f (t, y) - f (z, t) : sub_le_sub_right (candidates_triangle fA) _
+  f (x, y) - f(z, t) ≤ f (x, t) + f (t, y) - f (z, t) :
+      sub_le_sub_right (candidates_triangle fA) _
   ... ≤ (f (x, z) + f (z, t) + f(t, y)) - f (z, t) :
-    sub_le_sub_right (add_le_add_right (candidates_triangle fA) _ ) _
+      sub_le_sub_right (add_le_add_right (candidates_triangle fA) _ ) _
```

Comment:
In this large a PR, it's probably best not to also change whitespace in unrelated files

Reply:
I reverted these changes.  I had touched them earlier, when I had inadvertently changed a lemma name and did not think of undoing the formatting change, when the name changed back.